contracts/jobs/JobsManager.sol

```diff
@@ -233,15 +233,14 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
     }
 
     /*
-     * @dev Deposit funds for jobs
-     * @param _amount Amount to deposit
+     * @dev Deposit ETH for jobs
      */
-    function deposit(uint256 _amount) external whenSystemNotPaused {
-        broadcasters[msg.sender].deposit = broadcasters[msg.sender].deposit.add(_amount);
-        // Transfer tokens for deposit to Minter. Sender needs to approve amount first
-        livepeerToken().transferFrom(msg.sender, minter(), _amount);
+    function deposit() external payable whenSystemNotPaused {
+        broadcasters[msg.sender].deposit = broadcasters[msg.sender].deposit.add(msg.value);
+        // Transfer ETH for deposit to Minter
+        minter().receiveETH.value(msg.value)();
```

Comment:
Maybe name this function in the minter `deposit()` as well? Up to you. It feels a little weird to call a function called "receive"